opus/application/static_media/js/search.js

```diff
@@ -237,31 +237,31 @@ var o_search = {
             o_search.parseFinalNormalizedInputDataAndUpdateHash(slug, url);
         });
 
-        $('#search').on("change", 'input.multichoice, input.singlechoice', function() {
+        $("#search").on("change", "input.multichoice, input.singlechoice", function() {
            // mult widget gets changed
-           let id = $(this).attr("id").split('_')[0];
-           let value = $(this).attr("value").replace(/\+/g, '%2B');
-
-           if ($(this).is(':checked')) {
+           let id = $(this).attr("id").split("_")[0];
+           let value = $(this).attr("value").replace(/\+/g, "%2B");
```

Comment:
i'm sort of confused why there is a two-step replace/regexp here.  If it cannot be combined, perhaps a comment indicating what's going on would be useful.